src/core/xref.js

```diff
@@ -0,0 +1,134 @@
+// Automatically adds external refernces
+// https://github.com/w3c/respec/issues/1662
+
+import { norm } from "core/utils";
+import { pub } from "core/pubsubhub";
+
+/**
+ * main external reference driver
+ * @param {Object} conf respecConfig
+ * @param {Array:Elements} elems possibleExternalLinks
+ */
+export async function run(conf, elems) {
+  const xrefMap = createXrefMap(elems);
+
```

Comment:
nit: probably don't need this new line

Reply:
let me send the one with API end-point first.